postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -51,6 +56,8 @@
 
 EXPLAIN_VALIDATION_QUERY = "SELECT * FROM pg_stat_activity"
 
+MAX_ACTIVITY_QUERY_SIZE_QUERY = "SELECT setting FROM pg_settings WHERE name='track_activity_query_size'"
```

Comment:
Looks like the `pg_monitor` role grants access to read this so we should be good with the existing instructions for version 10+. Does this still work for 9.6 though?

Reply:
Looks like it passed CI so it must be working.

Reply:
> Looks like it passed CI so it must be working.

`:nod:`